docs/ecosystem/api-design.md

```diff
@@ -4,7 +4,7 @@ title: Design
 ---
 
 This document provides a summary of Ory's REST design with topics like pagination and date formats. If you're interested in Ory's
-API design, check out the [REST API design guidelines](../../open-source/guidelines/rest-api-guidelines).
+API design, check out the [REST API design guidelines](../open-source/guidelines/rest-api-guidelines.mdx).
```

Comment:
Relative linking with IDs instead of full filenames works flawlessly now - I wasn't able to reproduce any errors that you guys noticed a while back. Additionally, linking with document ID's allows to link to specific headings, whereas using full filenames doesn't if I'm correct. 

Also, this approach (linking through IDs) has been adapted across many of the new/revised documents so it'd be great to choose it as the target for consistency.

Reply:
Interesting, the docusaurus documentation seems to still prefer file paths: https://docusaurus.io/docs/markdown-features/links

Reply:
Specifically:


> Using relative file paths (with .md extensions) instead of relative URL links provides the following benefits:
>
> * Links will keep working on the GitHub interface and many Markdown editors
> * You can customize the files' slugs without having to update all the links
> * Moving files around the folders can be tracked by your editor, and some editors may automatically update file links
> * A [versioned doc](https://docusaurus.io/docs/versioning) will link to another doc of the exact same version
> * Relative URL links are very likely to break if you update the [trailingSlash config](https://docusaurus.io/docs/api/docusaurus-config#trailingSlash)

Reply:
I'm gonna reply in the issue you created.